### PR TITLE
Speed converter

### PR DESCRIPTION
**Pull request by JuanVT**
- Creating Speed converter form and choices
- Adding URL for speed converter
- Creating view for speed converter and its choices values
- Adding button for speed converter in base template

### CODE REVIEW

File: control/forms.py [hunk 1]
```diff
@@ -110,3 +110,20 @@ class WeightConverter(forms.Form):
...
     unit_value = forms.FloatField(initial=0)
+
+
+SPEED_CHOICES = [
+    ('', 'Select..'),
+    ('miles per hour', 'Miles per hour'),
+    ('foot per second', 'Foot per second'),
+    ('metre per second', 'Metre per second'),
+    ('kilometre per hour', 'Kilometre per hour'),
+    ('knot', 'knot'),
```
Comment (TysonRV) [inline control/forms.py]: Why is this one in lowercase?

File: control/forms.py [hunk 2]
```diff
@@ -110,3 +110,20 @@ class WeightConverter(forms.Form):
...
     unit_to = forms.ChoiceField(label='Unit to', choices=WEIGHT_CHOICES)
     unit_value = forms.FloatField(initial=0)
+
+
+SPEED_CHOICES = [
+    ('', 'Select..'),
+    ('miles per hour', 'Miles per hour'),
+    ('foot per second', 'Foot per second'),
+    ('metre per second', 'Metre per second'),
+    ('kilometre per hour', 'Kilometre per hour'),
```
Comment (TysonRV) [inline control/forms.py]: The values usually don't have spaces: `kilometre_per_hour`

File: control/views.py [hunk 3]
```diff
@@ -258,3 +258,36 @@ def weight_converter(request):
 
     template = loader.get_template('control/generic_converter.html')
     return HttpResponse(template.render(request=request, context=context))
+
+
+SPEED_VALUES = {
+    'miles per hour': 1,
+    'foot per second': 1.46667,
+    'metre per second': 0.44704,
+    'kilometre per hour': 1.60934,
```
Comment (TysonRV) [inline control/views.py]: Change the keys, replacing the spaces by `_`: `kilometre_per_hour`

File: control/forms.py (hunk 1 above)
Comment (JuanVT) [reply control/forms.py]: I might have missed it

File: control/forms.py (hunk 2 above)
Comment (JuanVT) [reply control/forms.py]: got it, will change for all of them. Thanks

File: control/views.py (hunk 3 above)
Comment (JuanVT) [reply control/views.py]: will do ;)

**Pull request merged**